module/applications/item/item-sheet.mjs

```diff
@@ -486,6 +490,28 @@ export default class ItemSheet5e extends ItemSheet {
     ];
   }
 
+  /* -------------------------------------------- */
+  /**
+   * Handle spawning the configuration applications.
+   * @param {Event} event   The click event which originated the selection.
+   * @private
```

Comment:
Now might not be the right time to start making this change, but I would vote for moving to formal private methods `#onConfigMenu` in any situation where we would be declaring the method as `@private` and reserve underscores for either `@internal` or `@protected` methods. The convention we now use in the core software.

Reply:
In this case I think this should probably be a `@protected` in case subclasses of item sheet want to extend it.